GitExtensions.settings

```diff
@@ -120,7 +120,7 @@
       &lt;RevisionPart&gt;LocalBranches&lt;/RevisionPart&gt;
       &lt;RevisionPart&gt;RemoteBranches&lt;/RevisionPart&gt;
     &lt;/SearchInParts&gt;
-    &lt;SearchPattern&gt;(?i)(?&amp;lt;!pull request |pr[ _]?)#\d+&lt;/SearchPattern&gt;
+    &lt;SearchPattern&gt;(?i)(?&amp;lt;!pull request |pr[ _]?)(#|(((feat(ure)?)|fix)[/_\-]))\d+&lt;/SearchPattern&gt;
```

Comment:
Do you want backslashes in the pattern, or is that esacping the hyphen? I don't think you have to escape hyphens when they're the last character in a `[ ]` construct. If you want `feat\1234` to match, you probably need `\\`.

Reply:
Thank you, you're right, the hyphen does not need to be escaped as last character.